modules/persistent-storage-csi-vsphere-install-issues.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+[id="persistent-storage-csi-vsphere-install-issues_{context}"]
+= Removing a non-Red Hat vSphere CSI Operator Driver
+
+If you have installed a non-Red Hat vSphere Container Storage Interface (CSI) Operator Driver, updates for your cluster to the next major version of {product-title}, such as 4.11, may be disabled due to {product-title} 4.10 having a built-in version of the vSphere CSI Operator Driver.
+
+{product-title} 4.10 clusters are still fully supported, and updates to bug fix versions of 4.10, such as 4.10.z, are not blocked, but you must correct this state by removing the non-Red Hat vSphere CSI Operator Driver before updates to next major version of {product-title} can occur. Removing the non-Red Hat vSphere CSI driver does not require deletion of associated persistent volume (PV) objects, and no data loss should occur.
+
+[NOTE]
+====
+These instructions may not be complete, so consult the vendor or community provider uninstall guide to ensure removal of the driver and components.
+====
+
+To uninstall the non-Red Hat vSphere CSI Operator Driver:
+
+. Delete the non-Red Hat vSphere CSI driver (VMware vSphere Container Storage Plug-in) Deployment and Daemonset objects.
+. Delete the configmap and secret objects that were installed previously with the non-Red Hat vSphere CSI Operator Driver.
```

Comment:
same here for `Operaror Driver`

Reply:
Implemented this comment.

Reply:
still says "Operator Driver"

Reply:
I don't know why that isn't updating. If you look at line 18 in the **File Changed** tab, it is correct.